proposals/xxxx-initializer-lists.md

```diff
@@ -0,0 +1,169 @@
+<!-- {% raw %} -->
+
+# Initializer Lists
+
+* Proposal: [NNNN](NNNN-initializer-lists.md)
+* Author(s): [Chris Bieneman](https://github.com/llvm-beanz)
+* Status: **Design In Progress**
+
+
+## Introduction
+
+HLSL's initializer lists assume a member-by-member flattened initialization
+sequence. It is significantly different in behavior from C/C++ initializer
+lists, but in a way that may not be apparent to users that they are depending on
+the divergent behavior. This proposal suggests an approach to implementing
```

Comment:
This line reads weird start with the forward looking  modifier something like this:
```
This proposal offers a forward-looking approach to implementing HLSL's unique initializer behavior.
```